reana_server/status.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Status module for REANA."""
+
+import subprocess
+from datetime import datetime, timedelta
+
+from invenio_accounts.models import SessionActivity
+from reana_commons.config import SHARED_VOLUME_PATH
+from reana_db.database import Session
+from reana_db.models import Workflow, WorkflowStatus
+
+
+class REANAStatus:
+    """REANA Status interface."""
+
+    def __init__(self, from_=None, until=None, user=None):
+        """Initialise REANAStatus class."""
+        self.from_ = from_ or (datetime.now() - timedelta(days=1))
+        self.until = until or datetime.now()
+        self.user = user
```

Comment:
Should we add a comment here indicating that they're not used yet but will be implemented? Otherwise might be confusing having those.